policy/approval/approve.go

```diff
@@ -102,6 +103,12 @@ func (r *Rule) Evaluate(ctx context.Context, prctx pull.Context) (res common.Res
 		res.Status = common.StatusApproved
 	} else {
 		res.Status = common.StatusPending
+
+		if r.Options.RequestReview {
+			// we don't use organizations, because that would be far too noisy
```

Comment:
Should this also be an option?

Reply:
I forget if this was clearly stated in the internal RFC or not, but the idea of the initial feature is that one member out of the potential reviewers is randomly selected to be assigned. I think both approaches with organizations (picking a random user or adding everyone) would likely be noisy in practice. 

That said, it might be more confusing to ignore organizations and we should rely on people to write policy rules that make sense.

Reply:
Do you mean one person _per rule_ that is pending, or one person _overall_? For now I've gone with the former, but its easy enough to change around.